src/Evolution/Systems/Cce/CMakeLists.txt

```diff
@@ -47,6 +47,7 @@ spectre_target_headers(
   SpecBoundaryData.hpp
   SwshDerivatives.hpp
   System.hpp
+  KleinGordonSystem.hpp
```

Comment:
It seems like this was forgotten in a previous PR, so move this to it's own commit